nemo_curator/modules/semantic_dedup/clusteringmodel.py

```diff
@@ -153,18 +165,22 @@ def __call__(self, embeddings_dataset: DocumentDataset):
             )
             cupy_darr.compute_chunk_sizes()
             t0 = time.time()
-            kmeans = KMeans(n_clusters=self.n_clusters, max_iter=self.max_iter)
+            kmeans = KMeans(
+                n_clusters=self.n_clusters,
+                max_iter=self.max_iter,
+                random_state=self.random_state,
+            )
```

Comment:
Just to confirm this is the main change right?

Reply:
Yes, everything else is docstring updates or updates to make sure all parameters are being used in the `scripts` directory. Thanks!